### PR TITLE
480-refactor: Remove css prefixes

### LINKED ISSUE #480: Investigate adding css prefix library
## 📝 Describe the feature

Investigate possibility of adding css prefix library in order to automate prefixing in our codebase

## ✅ Definition of Done (DoD):

1. Find good library for prefixing
2. Disable linter rule that promoting to add prefix
3. Remove all existed prefixes in our codebase and replace them with properties without the prefix
4. Ensure that the all prefixes applies correctly so we don't lose our styles using Playwright tests

### PR DESCRIPTION
**Pull request by Quiddlee**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [x] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Related Tickets & Documents

- Related Issue #480 
- Closes #480 

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

## Summary by CodeRabbit

- **New Features**
	- Enhanced accessibility of the Event Card by adding a `title` attribute to the additional information text.
  
- **Style**
	- Updated styling for the Event Card to prevent text wrapping, ensuring additional information truncates with an ellipsis if it exceeds the available space.

### CODE REVIEW

File: src/entities/event/ui/event-card/event-card.module.scss [hunk 1]
```diff
@@ -78,9 +78,6 @@
 
       .event-additional-info {
         overflow: hidden;
-        display: -webkit-box;
```
Comment (ansivgit) [inline src/entities/event/ui/event-card/event-card.module.scss:81]: We would like to limit additional info with 2 lines, so we need `line-clamp: 2`, but now it works only with prefix https://caniuse.com/?search=line-clamp

**Comment by Quiddlee** [comment]
> Should we also consider adding this rule? https://stylelint.io/user-guide/rules/property-no-vendor-prefix/

This rule is already applied

Screenshot 2024-11-21 at 10 53 46 PM

File: src/entities/event/ui/event-card/event-card.module.scss (hunk 1 above)
Comment (Quiddlee) [reply src/entities/event/ui/event-card/event-card.module.scss:81]: fixed

**Comment by SpaNb4** [comment]
> > Should we also consider adding this rule? https://stylelint.io/user-guide/rules/property-no-vendor-prefix/
> 
> This rule is already applied
> 
> Screenshot 2024-11-21 at 10 53 46 PM

Are you sure this is coming from stylelint and not your ide?

**Comment by Quiddlee** [comment]
> > > Should we also consider adding this rule? https://stylelint.io/user-guide/rules/property-no-vendor-prefix/
> > 
> > 
> > This rule is already applied
> > Screenshot 2024-11-21 at 10 53 46 PM
> 
> Are you sure this is coming from stylelint and not your ide?

it's using `stylelint` from `node_modules` and i believe configuration file from our project too

Screenshot 2024-11-21 at 11 55 30 PM

**Comment by Quiddlee** [comment]
> > > > Should we also consider adding this rule? https://stylelint.io/user-guide/rules/property-no-vendor-prefix/
> > > 
> > > 
> > > This rule is already applied
> > > Screenshot 2024-11-21 at 10 53 46 PM
> > 
> > 
> > Are you sure this is coming from stylelint and not your ide?
> 
> it's using `stylelint` from `node_modules` and i believe configuration file from our project too
> 
> Screenshot 2024-11-21 at 11 55 30 PM

it's somehow doesn't work if i use `npm run stylelint` even when explicitly define this rule

Screenshot 2024-11-22 at 12 06 33 AM

**Comment by SpaNb4** [comment]
> it's somehow doesn't work if i use `npm run stylelint` even when explicitly define this rule


that's weird, but okay
image

**Review by SpaNb4 — APPROVED** [review]

**Review by ansivgit — APPROVED** [review]

**Pull request merged**